packages/search/tests/integration/components/navi-search-result/definition-test.js

```diff
@@ -127,4 +119,119 @@ module('Integration | Component | definition', function(hooks) {
       'definitions sources are shown correctly'
     );
   });
+
+  test('result without a description', async function(assert) {
+    assert.expect(1);
+
+    const data = [
+      {
+        id: 'pageViews',
+        name: 'Page Views'
+      }
+    ];
+    set(this, 'data', data);
+
+    await render(hbs`<NaviSearchResult::Definition @data={{this.data}} />`);
+    assert.dom('.navi-search-definition-result').doesNotExist('No results are displayed');
```

Comment:
Can we add some tests for calling extended with and without a description

Reply:
yp added bellow!

Reply:
Can we add one more where extended does return a description?

Reply:
Oops, didn't look further above 👍